lib/logger.js

```diff
@@ -70,7 +70,8 @@ const streamConverter = {
         const impl = syslogStream.createBunyanStream({
             facility: stream.facility || 'local0',
             host: stream.host || '127.0.0.1',
-            port: stream.port || 514
+            port: stream.port || 514,
+            prefix: stream.prefix || ''
```

Comment:
The package also allows to set up the 'name'. Let's change this from explicit properties remapping to an `Object.assign` that would provide the defaults for the host, port and facility in order to allow additional configuration properties to be passed as is.

Reply:
Done